dotcom-rendering/src/components/WeatherWidget.tsx

```diff
@@ -171,50 +172,43 @@ const ExternalLinkIcon = () => (
 	</div>
 );
 
-export type TemperatureProps = {
-	value: number;
-	unit: 'C' | 'F';
-};
-
-type ForecastProps = {
-	icon: number;
-	description: string;
-	link: string;
-	temperature: {
-		metric: TemperatureProps;
-		imperial: TemperatureProps;
-	};
-	dateTime?: string;
-};
-
-export interface WeatherProps {
+export interface WeatherWidgetProps {
 	location: {
-		key: string;
-		localizedName: string;
-		country: {
-			id: string;
-			localizedName: string;
-		};
-		administrativeArea: {
-			id: string;
-			localizedName: string;
-		};
-		type: string;
+		id: string;
+		city: string;
+		country: string;
 	};
-	now: ForecastProps;
-	forecast: [ForecastProps, ForecastProps, ForecastProps, ForecastProps];
+	now: WeatherData;
+	forecast: WeatherForecast;
 	edition: FEFrontConfigType['edition'];
 }
 
-export const Weather = ({ location, now, forecast, edition }: WeatherProps) => {
+const getForecastForHourOffset = (
+	forecast: WeatherForecast,
+	offsetHours: 3 | 6 | 9 | 12,
+) => {
+	const dateTime = new Date(Date.now());
+	dateTime.setHours(dateTime.getHours() + offsetHours);
+	dateTime.setMinutes(0);
```

Comment:
Just double checking if this should be timezone aware?

Reply:
Ah crumbs, you're right.

I think Frontend is missing a field in its response to tell us what time these forecasts should be, I tried to workaround that but with timezones i don't think thats possible.